### PR TITLE
feat: support to replicate collection when the services contains the system tt msg

### PR DESCRIPTION
**Pull request by SimFG**
- issue: #37105

### CODE REVIEW

**Comment by SimFG** [comment]
rerun go-sdk

**Comment by SimFG** [comment]
rerun ut

**Comment by SimFG** [comment]
rerun ut

**Comment by SimFG** [comment]
rerun go-sdk

File: internal/proxy/task.go [hunk 1]
```diff
@@ -1081,6 +1082,29 @@ func (t *alterCollectionTask) PreExecute(ctx context.Context) error {
 		}
 	}
 
+	_, ok := common.IsReplicateEnabled(t.Properties)
+	if ok {
+		return merr.WrapErrParameterInvalidMsg("can't set the replicate.id property")
+	}
+	endTS, ok := common.GetReplicateEndTS(t.Properties)
+	if ok && collBasicInfo.replicateID != "" {
```
Comment (xiaofan-luan) [inline internal/proxy/task.go:1090]: add a param at AllocTimestampRequest and block wait at rootcoord.

So we don't need to sleep here

**Pull request merged**